tests/time_tests/test_runner/conftest.py

```diff
@@ -87,6 +87,41 @@ def pytest_addoption(parser):
         help='collection name in "{}" database'.format(DATABASE),
         choices=DB_COLLECTIONS
     )
+    db_args_parser.addoption(
+        '--os',
+        type=str,
+        required=is_db_used,
+        help='os name and version',
+        choices=["ubuntu_18_04", "ubuntu_20_04", "windows_vs2019"]
+    )
+    db_args_parser.addoption(
+        '--arch',
+        type=str,
+        required=is_db_used,
+        help='system architecture',
+        choices=["intel64", "ia32"]
+    )
+    db_args_parser.addoption(
+        '--repo_url',
+        type=str,
+        metavar="URL",
+        required=is_db_used,
+        help='URL of repository where changes caused tests run. For local use specify OpenVINO URL.'
+    )
+    db_args_parser.addoption(
+        '--commit',
+        type=str,
+        nargs=3,
+        metavar="BRANCH,SHA,DATE",
```

Comment:
We should take that data from manifest. Manifest travels with the build and should be single point of information. Tool, optionally, might support workarounds to supply necessary information directly

Reply:
Done using `--manifest` and `--db_metadata` CLI keys (`--metadata` can't be used because of conflict with `pytest-metadata` plugin)